### PR TITLE
Release_management

### PR DESCRIPTION
**Pull request by pcalatayud-prog**
Release management: a specific Kanban project for each release,

### CODE REVIEW

**Review by orviz — CHANGES_REQUESTED** [review]
Remove first line & set "Release policy" as a markdown section

**Review by Yosoycoder — APPROVED** [review]
I approve the contribution although there are some requested changes pending.

File: Release_management.md [hunk 1]
```diff
@@ -1,18 +1,15 @@
-Release management: a specific Kanban project for each release, follow semver @pcalatayud-prog
+#Release policy.
```
Comment (Yosoycoder) [inline Release_management.md:1]: Add space after #

File: Release_management.md [hunk 2]
```diff
@@ -1,18 +1,15 @@
...
 
+##1. Desing a kanban project from scratch to the first release and a new one between releases.
```
Comment (Yosoycoder) [inline Release_management.md:3]: Add space after ##

File: Release_management.md [hunk 3]
```diff
@@ -1,18 +1,15 @@
...
 
-Release policy.
-
-1. Desing a kanban project from scratch to the first release and a new one between releases.
-
-2. First release version starts from 1.0.0.
+##2. First release version starts from 1.0.0.
```
Comment (Yosoycoder) [inline Release_management.md:5]: Add space after ##

File: Release_management.md [hunk 4]
```diff
@@ -1,18 +1,15 @@
...
 	2.1. Semantic Versionin standard is followed (x.y.z).
 		2.1.1 x is a mayor release: has changes that may be incompatible with prior major releases.
 		2.1.2 y is a minor release: adds new functionality and bug fixes in a backwards
 		compatible manner.
 		2.1.3 Z is a patch release: adds backwards compatible bug fixes.
 
-3. kanban is composed by 5 columns:
+##3. kanban is composed by 5 columns:
```
Comment (Yosoycoder) [inline Release_management.md:12]: Add space after ##

**Review by Yosoycoder — CHANGES_REQUESTED** [review]

**Review by Yosoycoder — APPROVED** [review]
OK

**Review by orviz — APPROVED** [review]

**Review by DashaEscobar — APPROVED** [review]

**Pull request merged**